### ISSUE TITLE
"More tweets" appearing despite them being turned off

### ISSUE THREAD

**Issue by Absolucy** [insin/tweak-new-twitter#97]
Browser: MS Edge

(sorry for oversaturation, bug with sharex)
![image](https://user-images.githubusercontent.com/65794972/155048540-edee6f4e-2bc3-4e5a-95a7-46edc3db9265.png)

**Comment by BarnMTB** [comment]
I also have this issue for a while now and it still is an issue. (Using Edge)

This happens when I open a link to the tweet into new tab and that link has a tracking parameters attached to it.

For example:
https://twitter.com/windowscentral/status/1557740721515286528?s=20&t=lrEFw2Fy5VJdlftld_byhw will show More tweets,

But "More tweets" will be hidden as intended if the link was this link instead https://twitter.com/windowscentral/status/1557740721515286528 

I've tested this by opening a new tab and pasting each tracked-link and clean-link in it.
I've tried this on both Edge and Firefox, and Firefox seems to not have this issue. 
(Note: I have ClearURLs installed, enabled, and configured identically on both browsers.)
